### PR TITLE
XW-117 | Remove Venus-related extensions

### PR DESCRIPTION
**Pull request by vforge**
Remove: 
- Zenbox
- LocalNavigation
- ArticleNavigation
- Recommendations (and it's API)

/edit
- Venus
- VenusTest

see https://github.com/Wikia/app/pull/8656

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by bognix** [comment]
LGTM

File: extensions/wikia/NjordPrototype/css/icons.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@charset "UTF-8";
+
+@font-face {
+	font-family: 'njord-icons-font';
+	src: url('/extensions/wikia/NjordPrototype/css/fonts/venus-icons.eot'); /* $wgCdnStylePath */
```
Comment (Warkot) [inline extensions/wikia/NjordPrototype/css/icons.scss:5]: Should all of these still be called `venus-*`?

File: extensions/wikia/RecentWikiActivity/RecentWikiActivity.i18n.php [hunk 2]
```diff
@@ -1,294 +0,0 @@
-<?php
```
Comment (Warkot) [inline extensions/wikia/RecentWikiActivity/RecentWikiActivity.i18n.php:1]: Isn't Oasis using RWA?

Comment (vforge) [inline extensions/wikia/RecentWikiActivity/RecentWikiActivity.i18n.php:1]: Nope

File: extensions/wikia/NjordPrototype/css/icons.scss (hunk 1 above)
Comment (vforge) [inline extensions/wikia/NjordPrototype/css/icons.scss:5]: Talked offline

**Comment by bognix** [comment]
LGTM

**Comment by macbre** [comment]
Ladies and gentlemen, please remember to check i18n files references in config when making changes in app - see https://github.com/Wikia/config/pull/1482

**Comment by vforge** [comment]
@macbre Sorry about that.

**Comment by macbre** [comment]
No worries :) Cleanup performed.

**Pull request merged**